packages/rbflabs-agoric-react-components/src/contexts/wallet/hooks/use-wallet-notifications/use-wallet-notifications.ts

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useRef } from 'react';
+import ShortUniqueId from 'short-unique-id';
+import { usePrevious } from '../../../../hooks';
+import { keys } from '../../../../utils/object';
+import { WalletNotification, WalletNotificationGroup } from '../../types';
+import { useWallet } from '../use-wallet';
+import { getConnectNotification, getOfferNotification, getUpdatedOffer } from './utils';
+
+const uid = new ShortUniqueId({ length: 10 });
```

Comment:
Wouldn't it be better to use [Crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) instead of adding another dependency?

Reply:
Sure, did not know about this until now, thanks.